spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/AbstractConfigProperties.java

```diff
@@ -21,7 +21,7 @@
  *
  * @author Ioannis Canellos
  */
-public abstract class AbstractConfigProperties {
```

Comment:
can lower the visibility as this class is not used anywhere outside the package

Reply:
I dont want to do this now that it was already public, technically it could break someone.